dask/dataframe/csv.py

```diff
@@ -0,0 +1,142 @@
+from __future__ import print_function, division, absolute_import
+
+from io import BytesIO
+from warnings import warn
+
+import numpy as np
+import pandas as pd
+
+from ..delayed import delayed
+from .io import from_delayed
+
+from ..bytes import read_bytes
+from ..bytes.compression import seekable_files, files as cfiles
+from ..utils import ensure_bytes
+
+
+delayed = delayed(pure=True)
+
+
+def bytes_read_csv(b, header, kwargs, dtypes=None, columns=None):
```

Comment:
Not really, no.  I imagine other modules like `csv.py` being built independently and making use of functions like `read_bytes`.  

Of those three formats I suspect that only avro is appropriate for the `read_bytes` treatment.  Feather isn't intended to be split.  Parquet's layout means that different blocks have very different meaning.